### ISSUE TITLE
Mod requests java 21 on minecraft 1.20.1 which uses java 17

### ISSUE THREAD

**Issue by vortivoid** [T0RNATO/drop-spread-2#1]
The 1.20.1 version of this mod doesn't launch. The error message seems to hint that the mod wants java 21, despite the fact that minecraft 1.20.1 uses java 17, and won't launch if you use 21.

I'm using prismlauncher, though I doubt that makes a difference.

![Screenshot from 2024-10-16 01-35-23](https://github.com/user-attachments/assets/f563bf78-b6b4-4a45-9615-160a70742c5a)

**Comment by T0RNATO** [comment]
Yeah, I had some incorrect dependencies, but pretty sure I fixed it in 1.0.1 - what version of the mod are you on?

**Comment by vortivoid** [comment]
Can confirm i'm using 1.0.1 (or at least, modrinth displays the version as 1.0.1 however the mod itself displays as 1.0.0, i'm unsure if that's just an error made during upload but might be important?)
![Screenshot from 2024-10-16 01-47-37](https://github.com/user-attachments/assets/fe3e896a-f696-489f-b4fb-03d2e22b35ab)
![Screenshot from 2024-10-16 01-48-02](https://github.com/user-attachments/assets/46e93136-48aa-4b27-884c-ff9da29a52a2)

**Comment by T0RNATO** [comment]
Hmm, I'll take a look at it when I get home.

**Comment by vortivoid** [comment]
Okay interesting development, I tried launching it on my server and it seemed to work perfectly fine there? I'm unsure now if it's just an issue with my desktop device, or if the mod only works on servers for some reason? I'll have to check on a different device, I'll report back once i've gotten round to that (hopefully shortly)

**Comment by vortivoid** [comment]
Okay i've checked on a different device, and i can safely say it breaks on the client, but works fine on servers.

**Comment by T0RNATO** [comment]
Okay so I have no idea why this is the case - it literally only depends on FAPI (which I'm not even sure if it requires tbh, lol)
![image](https://github.com/user-attachments/assets/0734dcb1-a8a9-4d05-8fb9-f39d73bde1f0)
(Also, sorry for the late reply, I've just been busy.)
Maybe it's the java section in the build.gradle. I'll check it out.

**Comment by T0RNATO** [comment]
@Seppy-Dev Hopefully this works.
https://modrinth.com/mod/drop-spread-2/version/6O9B1B2V
I did all my testing on all the versions with Java 21, oops.

**Comment by vortivoid** [comment]
Still same issue i'm afriad. Yet again tested on a seperate device to make sure.

Here's the full log: https://mclo.gs/xV96wkb

**Comment by T0RNATO** [comment]
*Hopefully*, this should work for you.
https://modrinth.com/mod/drop-spread-2/version/UfgF5C8v
Thanks for your patience.

**Comment by NotRyken** [comment]
Issue persists on v1.0.3 https://mclo.gs/saKhYFg

**Comment by T0RNATO** [comment]
oh I have no idea why this is happening, I'll try and fix it over the next few days.

**Comment by T0RNATO** [comment]
*Theoretically* this is now finally working. Please test. https://modrinth.com/mod/drop-spread-2/version/Ezm2ZepP
@Seppy-Dev

**Comment by T0RNATO** [comment]
Closing stale issue.
